src/authentication/auth-config.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
```

Comment:
Will kept the name of this file as aws-exports.js do you want to keep it the same as him ?

Reply:
changed